### PR TITLE
Libra-Gaussian update

### PR DESCRIPTION
**Pull request by Pulokesh**

### CODE REVIEW

File: src/detect_g09.py [hunk 1]
```diff
@@ -0,0 +1,228 @@
+#*********************************************************************************
```
Comment (alexvakimov) [inline src/detect_g09.py]: wasn't this file already included into the repository? why does it appear as the entire file is a new commit? this seems improper use of git

File: src/extract_g09.py [hunk 2]
```diff
@@ -0,0 +1,445 @@
+#*********************************************************************************
```
Comment (alexvakimov) [inline src/extract_g09.py]: same with this file

File: src/md.py [hunk 3]
```diff
@@ -346,24 +346,24 @@ def run_MD(syst,el,ao,E,sd_basis,params,label,Q, active_space):
...
                         # Compose electronic and vibronic Hamiltonians
-                        #t.stop()
-                        #print "time before update vib ham=",t.show(),"sec"
-                        #if params["non-orth"] ==1:
-                        #    vibronic_hamiltonian_non_orth(ham_adi[cnt], ham_vib[cnt], params, E_SD_old,E_SD,nac,smat_old,smat, str(ij))
-                        #else:
-                        #    update_vibronic_hamiltonian(ham_adi[cnt], ham_vib[cnt], params, E_SD,nac, str(ij), opt)
-                        #t.stop()
-                        #print "time after update vib ham=",t.show(),"sec"
+                        t.stop()
```
Comment (alexvakimov) [inline src/md.py]: why do I constantly see this part of the code updated?

File: src/x_to_libra_g09.py [hunk 4]
```diff
@@ -0,0 +1,170 @@
+#*********************************************************************************
```
Comment (alexvakimov) [inline src/x_to_libra_g09.py]: same with this file.. - use the existing file, don't create a new copy of the file

File: src/detect_g09.py (hunk 1 above)
Comment (Pulokesh) [inline src/detect_g09.py]: I don't know, I am still in the old Libra-X, not cloned it from Quantum-Dynamics-Hub. I think this is the reason why it is showing the entire file in the new commit.

**Pull request closed without merging**